.github/workflows/ScenarioTest.yaml

```diff
@@ -91,104 +91,104 @@ jobs:
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=momentary_stop osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=20 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run reaction_time scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time timeout:=10 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=reaction_time osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=10 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run out_of_range scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=out_of_range timeout:=7 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=out_of_range osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=7 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run stop_at_crosswalk scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=stop_at_crosswalk timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=stop_at_crosswalk map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run move_backward scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=move_backward timeout:=10 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=move_backward osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=10 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run get_longitudinal_distance scenario
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=get_longitudinal_distance timeout:=15 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=get_longitudinal_distance map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```

Comment:
.osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.